### DISCUSSION TITLE
Method exclusion

### DISCUSSION THREAD

**Discussion by Gonaf** [codewars/codewars.com#2988] [General]
Hello, I'm fairly new on creating my own kata. Im trying to make a probleminvolving Strings that is really simple to solve with regex, but I want it to be solved by looping through a given String, is there a way to prevent users to call any regex methods?

**Comment by HartlIKS** [comment]
In most languages you can easily look at the text that the user submitted as the solution.
It's usually in a file called solution.txt in the working directory in which the tests are executed.
In your submission tests you can search that text file for stuff that shouldn't be there. (e.g. references to regexp functions).

In some languages you can overwrite library functions. In those cases you can simply overwrite the regexp stuff with null/none/undefined/whatever.
